crates/router/src/configs/settings.rs

```diff
@@ -128,6 +128,7 @@ pub struct Settings<S: SecretState> {
     pub network_tokenization_supported_card_networks: NetworkTokenizationSupportedCardNetworks,
     pub network_tokenization_service: Option<SecretStateContainer<NetworkTokenizationService, S>>,
     pub network_tokenization_supported_connectors: NetworkTokenizationSupportedConnectors,
+    pub theme_storage: FileStorageConfig,
```

Comment:
> I don't want to use the file storage bucket for this. [...]

I was asking from the perspective of users self-hosting Hyperswitch, they may choose to use a single bucket because of policies within their organization, etc.

Reply:
Yeah, they can with this setup.